GameData/WildBlueIndustries/000WildBlueTools/Templates/ClassicStock/Storage/Hydrokarbon.txt

```diff
@@ -0,0 +1,354 @@
+RESOURCE_DEFINITION
+{
+	name = Hydrokarbon // LqdMethane
+	displayName = Hydrokarbon
+	abbreviation = HK
+	density = 0.00042561
+	unitCost = 0.05
+	hsp = 3348.8
+	vsp = 510000
+	flowMode = STACK_PRIORITY_SEARCH
+	transfer = PUMP
+	isTweakable = True
+	isVisible = True
+	color = 0.93, 0.87, 0.38
+	volume = 1
+}
+
+RESOURCE_DEFINITION
+{
+	name = RHK1 // Karbonite
+	displayName = RHK-1
+	density = 0.0025
+	unitCost = 0.32
+	flowMode = STACK_PRIORITY_SEARCH
+	transfer = PUMP
+	isTweakable = True
+	isVisible = True
+	volume = 1
+}
+
+STORAGE_TEMPLATE
+{
+	name = Hydrokarbon
+	author = JadeOfMaar
+	shortName = Hydrokarbon
```

Comment:
Short names are no longer needed. Please use title instead if you need a name that's not the actual resource name.